GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -81,20 +81,23 @@ extension Glia {
 
     func companyName(
         using interactor: Interactor,
-        currentName: String?
+        themeCompanyName: String?
     ) -> String {
-        // As the default value is empty, it means that the integrator
-        // has set a value on the theme itself. Return that same value.
-        if let currentName, !currentName.isEmpty {
-            return currentName
-        }
-
         let companyNameStringKey = "general.company_name"
 
-        // Company name has been set on the custom locale.
-        if let remoteCompanyName = stringProviding?.getRemoteString(companyNameStringKey) {
+        // Company name has been set on the custom locale and is not empty.
+        if
+            let remoteCompanyName = stringProviding?.getRemoteString(companyNameStringKey),
+            !remoteCompanyName.isEmpty
+        {
```

Comment:
didn't see this kind of formatting in our code base, would it make sense to align with other code or provide a new codebase style?

Reply:
Fixed beb7772ce9aeb5f947e11b96840126b360471bf5